assets/scss/components/_update-goal.scss

```diff
@@ -0,0 +1,7 @@
+.step-status-radios .govuk-label{
+  max-width: 100%;
+}
```

Comment:
the small radios + inline radios GDS classes together caused some wrapping with "Not started" which this fixes. I tried it at a number of screen sizes and it seems to wrap correctly still.